### PR TITLE
Add installation instructions

### PR DESCRIPTION
**Pull request by dvdksn**
Signed-off-by: David Karlsson <35727626+dvdksn@users.noreply.github.com>

### CODE REVIEW

**Comment by dvdksn** [comment]
Thanks @sgarcez , can you merge this too? I don't have the permissions

**Comment by dvdksn** [comment]
should we merge this @sgarcez ?

File: README.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
 # buildx-desktop
 
 This repo contains buildx-desktop releases.
+
+## Install
+
+1. Download the latest version of the binary matching your OS and architecture
+   from the [Releases](https://github.com/docker/buildx-desktop/releases) page.
+
+2. Move the binary to `~/.docker/cli-plugins/docker-buildx`
```
Comment (sgarcez) [inline README.md:10]:
```suggestion
2. Move the binary to `~/.docker/cli-plugins/docker-buildx`
3. Change its permissions `chmod a+x ~/.docker/cli-plugins/docker-buildx`
```

**Comment by dvdksn** [comment]
ping @sgarcez

**Pull request merged**